kernel/prepare/pkg.yaml

```diff
@@ -11,42 +11,20 @@ steps:
         sha512: "{{ .linux_sha512 }}"
     env:
       ARCH: {{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}x86_64{{ else }}unsupported{{ end }}
+  - network: default
     prepare:
       - |
         tar -xJf linux.tar.xz --strip-components=1
         rm linux.tar.xz
 
-        mkdir /bin
-        ln -sv /toolchain/bin/bash /bin/bash
-        ln -sv /toolchain/bin/bash /bin/sh
-        ln -sv /toolchain/lib /lib
-        mkdir -p /usr/bin \
-            && ln -sf /toolchain/bin/env /usr/bin/env \
-            && ln -sf /toolchain/bin/true /bin/true \
-            && ln -sf /toolchain/bin/false /bin/false \
-            && ln -sf /toolchain/bin/pwd /bin/pwd
-
-        # Ensure that `make menuconfig` works.
-        ln -s /toolchain/bin/awk /usr/bin/awk
-
-        mkdir -p /usr/lib/pkgconfig
-        ln -s /toolchain/include /usr/include
-
         make mrproper
       - |
-        cd /toolchain && git clone https://github.com/a13xp0p0v/kernel-hardening-checker.git
+        # TODO: remove
+        mkdir /toolchain && cd /toolchain && git clone https://github.com/a13xp0p0v/kernel-hardening-checker.git
```

Comment:
Why do we pull this with git and not just as a tar?

Reply:
No good reason, we want the latest & greatest, and it doesn't go into the build results, it's a linter.

Reply:
`wget https://github.com/a13xp0p0v/kernel-hardening-checker/archive/refs/heads/master.tar.gz` seems to work. Then git can perhaps be removed from tools

Reply:
Done using pulling from master. However I checked and this tool runs in the build step (before artifact creation). So if it's not pinned and gets replaced by malware it could affect the build. Also makes building older versions not guaranteed to succeed, because if some rule is added to KSPP and someone wants to build an old version, it won't let it work. Anyway, keeping mostly the same but using wget now